app/views/dashboard/show.html.erb

```diff
@@ -1,27 +1,5 @@
-<div class="govuk-grid-row govuk-!-margin-bottom-6">
-  <div class="govuk-grid-column-full">
-    <div class="blue-header">
-      <h1 class="govuk-heading-xl blue-header__title">Welcome <%= current_user.name %></h1>
-      <%= govuk_link_to "Edit name", edit_preferred_name_path, class: "blue-header__text", "data-test": "edit-name" %>
-    </div>
-
-    <% if current_user.early_career_teacher? %>
-      <%= govuk_link_to "Go to your module materials",
-        cip_path(current_user.early_career_teacher_profile.core_induction_programme),
-        button: true %>
-    <% end %>
-
-    <% if current_user.mentor? && current_user.mentor_profile.core_induction_programme %>
-      <%= govuk_link_to "Go to your mentee's module materials",
-        cip_path(current_user.mentor_profile.core_induction_programme),
-        button: true %>
-    <% end %>
-
-    <h2 class="govuk-heading-l govuk-!-margin-top-4">Related content</h2>
-
-    <p><%= govuk_link_to "Information about training and support during your induction", training_and_support_path %></p>
-    <p><%= govuk_link_to "How the teacher standards relate to the ECF", "#" %></p>
-    <p><%= govuk_link_to "Your mentor's role and responsibilities", "#" %></p>
-
-  </div>
-</div>
+<% if current_user.mentor? %>
+  <%= render "dashboard_mentor" %>
+<% else %>
+  <%= render "dashboard_ect" %>
+<% end %>
```

Comment:
Can we have a check that user is an ect before we show them ect dashboard, and maybe have a `default_dashboard` shown otherwise?

Reply:
Also, just to double check, are we absolutely sure that we want to make dashboards look very different for different users?

Reply:
i'm kind of hoping the ECT dashboard changes to match the mentor dashboard, but yes, they're currently very different in the designs

I don't think it's really worth the effort of making a default dashboard unless there's something specific that would be gained from it - I don't think there's any problem with showing them the ECT dashboard